fe1-web/__tests__/model/objects/WalletCryptographyHandler.test.ts

```diff
@@ -1,51 +1,11 @@
 import 'jest-extended';
 import { WalletCryptographyHandler } from 'model/objects/WalletCryptographyHandler';
 import { sign } from 'tweetnacl';
-import STRINGS from 'res/strings';
-import { get } from 'idb-keyval';
+import { WalletCryptoLibrary } from 'res/adhoc-implementations/__mocks__/WalletCryptoLibrary';
 
 /* used to simulate indexedDB database to test store/retrieve functions */
 require('fake-indexeddb/auto');
```

Comment:
you should be able to do 
```suggestion
import 'fake-indexeddb/auto';
```